README.md

```diff
@@ -34,38 +31,36 @@ support is required.
 * The `Response` class for response objects can be substituted by a custom
   response class.
 
-
 ### Limitations
 
-- `mrequests.request` is a synchroneous, blocking function.
-- The code is *not* interrupt save and a fair amount of memory allocation is
+* `mrequests.request` is a synchroneous, blocking function.
```

Comment:
Normally I don't like mixing feature and clean-up commits, but I guess I can accept them this time.

Reply:
this was also added by VS Code on save 😭 

The markdown linter for VS Code uses these customs

```
* first level
  + second level
    - third level
```

I can revert these and add a cleanup PR if that suits you :)

Reply:
No, leave those in. Like I said, it's ok in this instance.